### DISCUSSION TITLE
Edge Functions Unable to connect (Type Error)

### DISCUSSION THREAD

**Discussion by LinThitHtwe** [supabase/supabase#27667] [Questions]
I've got this problem after not running the project for a while and I've no idea how to solve this. I don't implement any edge functions, it just plain database with tables and nothing else.

My project is with NextJs. 
So when I make a request, I got a Type Error.

![EdgeFunction](https://github.com/supabase/supabase/assets/106507721/07a9cb9b-4f06-44e4-a4f0-9479123cce5b)

**Comment by silentworks** [comment]
You should contact Supabase support about his. https://supabase.com/dashboard/support/new
